specification/cognitiveservices/data-plane/Face/preview/v1.0-preview/Face.json

```diff
@@ -4813,6 +4861,24 @@
       "minimum": 0,
       "maximum": 1
     },
+    "NonNullableNameAndNullableUserDataContract": {
```

Comment:
What is the rationale for this name? This seems like it will translate into a poor UX.

Reply:
Mostly for internal disambiguation, as the struct is only consumed via the "allOf" in MetaDataContract.

Reply:
None of the properties in the model are nullable - it has an optional `userDataContract` field. I would strongly recommend trying to find a better name for this.